### PR TITLE
https://issues.redhat.com/browse/ACM-6844  FIPS mode or the cluster i…

### PR DESCRIPTION
**Pull request by swopebe**
…s in disconnected environment.

### CODE REVIEW

File: clusters/hosted_control_planes/configure_hosted_aws.adoc [hunk 1]
```diff
@@ -3,13 +3,16 @@
 
 To configure hosted control planes, you need a hosting cluster and a hosted cluster. By deploying the HyperShift Operator on an existing managed cluster by using the `hypershift-addon` managed cluster add-on, you can enable that cluster as a hosting cluster and start to create the hosted cluster. 
 
-The {mce-short} 2.2 supports only the default `local-cluster` and the hub cluster as the hosting cluster.
+*Important:* Do not enable this feature if the cluster is configured for FIPS mode or the cluster is in disconnected environment.
+
+The {mce-short} 2.3 supports only the default `local-cluster` and the hub cluster as the hosting cluster.
 
 Hosted control planes is a Technology Preview feature, so the related components are disabled by default.
+// does this change?
```
Comment (rokej) [inline clusters/hosted_control_planes/configure_hosted_aws.adoc]: `Hosted control planes is a Technology Preview feature, so the related components are disabled by default.` is still true for MCE 2.3, ACM 2.8

**Review by rokej — APPROVED** [review]

File: clusters/hosted_control_planes/configure_hosted_aws.adoc [hunk 2]
```diff
@@ -3,13 +3,15 @@
 
 To configure hosted control planes, you need a hosting cluster and a hosted cluster. By deploying the HyperShift Operator on an existing managed cluster by using the `hypershift-addon` managed cluster add-on, you can enable that cluster as a hosting cluster and start to create the hosted cluster. 
 
-The {mce-short} 2.2 supports only the default `local-cluster` and the hub cluster as the hosting cluster.
+*Important:* Do not enable this feature if the cluster is configured for FIPS mode or the cluster is in disconnected environment.
```
Comment (swopebe) [inline clusters/hosted_control_planes/configure_hosted_aws.adoc]: New content on line 6.

Comment (lahinson) [inline clusters/hosted_control_planes/configure_hosted_aws.adoc]:
```suggestion
*Important:* Do not enable this feature if the cluster is configured for FIPS mode or the cluster is in a disconnected environment.
```

**Review by lahinson — COMMENTED** [review]
@swopebe I made one tiny suggestion. Otherwise, LGTM!

**Review by lahinson — APPROVED** [review]

**Pull request merged**